velox/docs/functions/spark/decimal.rst

```diff
@@ -43,6 +46,26 @@ precision and scale are adjusted.
     precision = 38
     scale = max(38 - (p - s), min(s, 6))
 
+Caps p and s at 38 when not allowing precision loss.
+For decimal addition, subtraction, multiplication, the precision and scale computation logic is same,
+but for decimal division, it is different as following:
+::
+
+    wholeDigits = min(38, p1 - s1 + s2);
+    fractionalDigits = min(38, max(6, s1 + p2 + 1));
```

Comment:
The `Decimal Precision and Scale Adjustment` section discusses how to handle the case when the calculated precision exceeds 38, so I assume this part and L66 belong elsewhere (L33 I think). Perhaps only keep the description for precision and scale adjustment at L57.

> For decimal addition, subtraction, multiplication, the precision and scale computation logic is same

This can be described in the corresponding location, such as L12 or L20.